### PR TITLE
Improved commit messages.

### PR DESCRIPTION
**Pull request by JAremko**
Some commits needed a better message.

### CODE REVIEW

**Comment by joehillen** [comment]
WTF?

**Comment by JAremko** [comment]
Damn conflicts. Can't solve em :confounded:

**Comment by robbyoconnor** [comment]
@JAremko  is amazing.

**Comment by d12frosted** [comment]
@robbyoconnor :+1: amazing indeed

On Sat, Apr 2, 2016 at 4:29 AM, robbyoconnor <notifications@github.com>
wrote:

> @JAremko <https://github.com/JAremko> is amazing.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/5653#issuecomment-204623905>
>



-- 
~ d12frosted

**Comment by robbyoconnor** [comment]
This pull request will make everyone unhappy

File: layers/+tools/pdf-tools/README.org [hunk 1]
```diff
@@ -18,7 +18,7 @@
    - [[Occur mode][Occur mode]]
 
 * Description
-This layer enables support for the [[https://github.com/politza/pdf-tools][=pdf-tools=]] package.
+This layer enables support for the [[https://github.com/politza/pdf-tools][pdf-tools]] package.
```
Comment (robbyoconnor) [inline layers/+tools/pdf-tools/README.org]: leave it -- it's actually valid.

**Comment by JAremko** [comment]
Closing it before someone gets heart attack.  :smile:

**Comment by robbyoconnor** [comment]
Literally will make anyone who has an open PR's life HELL -- and i mean HELL -- not the party destination -- I'm talking down in the lowest part of it!

**Pull request closed without merging**